ui/src/main/webapp/src/app/configuration/configuration.component.html

```diff
@@ -30,9 +30,9 @@ <h1 i18n="Header" style="margin-bottom: 25px">Rules Configuration</h1>
             <!-- Header for each set of rules. -->
             <div class="panel-heading">
                 <h2 class="panel-title">
-                    <span style="float: right" class="pointer" (click)="confirmRemoveRules(rulePath)" *ngIf="rulePath.rulesPathType != 'SYSTEM_PROVIDED'" title="Delete">
-                        <span class="glyphicon glyphicon-trash"></span>
-                    </span>
+                    <button class="btn btn-default pull-right" (click)="confirmRemoveRules(rulePath)" *ngIf="rulePath.rulesPathType != 'SYSTEM_PROVIDED'">
+                        <i class="glyphicon glyphicon-trash"></i> <!--Delete-->
+                    </button>
```

Comment:
this is from the solution before, I tried to disable the clicking, but span doesn't have the disabled property and with current solution I forgot to change it back ;)